app/views/shared/_navbar.html.erb

```diff
@@ -29,6 +29,8 @@
                 <%= link_to "Outreach Event Types", admin_outreach_event_types_path, class: "navbar-item" %>
               </div>
             </div>
+          <% elsif current_user.facilitator? %>
+            <%= link_to "Outreach Events", facilitator_outreach_events_path, class: "navbar-item" %>
```

Comment:
This shouldn't be necessary if we remove the facilitator's outreach events route